versioned_docs/version-1.3.0-beta/client_libraries/go.mdx

```diff
@@ -10,46 +10,37 @@ Source code: [github.com/oras-project/oras-go](https://github.com/oras-project/o
 ## Introduction
 
 The ORAS Go client library provides the ability to replicate artifacts between different [Targets](./overview.mdx#target).  
-Furthermore, the version `v2` is a registry client conforming [image-spec v1.1.0-rc6](https://github.com/opencontainers/image-spec/releases/tag/v1.1.0-rc6) and [distribution-spec v1.1.0-rc4](https://github.com/opencontainers/distribution-spec/releases/tag/v1.1.0-rc4).
+Furthermore, the version `v2` is a registry client conforming [image-spec 1.1.1](https://github.com/opencontainers/image-spec/releases/tag/v1.1.1) and [distribution-spec v1.1.1](https://github.com/opencontainers/distribution-spec/releases/tag/v1.1.1).
 
 Using the ORAS Go client library, you can develop your own registry client:
 
 ```sh
 myclient push artifacts.example.com/myartifact:1.0 ./mything.thang
 ```
 
-## Usage
+## Getting Started
 
-The package `oras.land/oras-go/v2` can quickly be imported in other Go-based tools that
-wish to benefit from the ability to store arbitrary content in container registries.
+### Concepts
 
-1. Get the  `oras.land/oras-go/v2` package
-```sh
-go get oras.land/oras-go/v2
-```
+Gain insights into the fundamental concepts:
 
-2. Import and use the `v2` package
-```go
-import "oras.land/oras-go/v2"
-```
+- [Modeling Artifacts](https://github.com/oras-project/oras-go/blob/main/docs/Modeling-Artifacts.md)
+- [Targets and Content Stores](https://github.com/oras-project/oras-go/blob/main/docs/Targets.md)
 
-3. Run
-```sh
-go mod tidy
-```
+### Quickstart
 
-**The API documentation and examples are available at [pkg.go.dev](https://pkg.go.dev/oras.land/oras-go/v2).**
+Follow the step-by-step tutorial to use `oras-go` v2:
 
-## Quick Start
+- [Quickstart: Managing OCI Artifacts with `oras-go` v2](https://github.com/oras-project/oras-go/blob/main/docs/tutorial/quickstart.md)
```

Comment:
Will it be better to use a tagged version instead of `main` branch?

Reply:
+1

Reply:
Updated. Used the latest tag `v2.6.0`.